.github/workflows/jekyll.yaml

```diff
@@ -0,0 +1,57 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+name: CI
+
+on:
```

Comment:
I presume that keeping the branch name as `main` is to avoid publishing in-flight changes from other PRs from "tainting" the live website, correct?

Reply:
The built site in the other branches isn't itself buildable in any way, so it doesn't make sense to execute any CI builds on those branches, whether or not somebody tries to make a PR against them (which we don't expect, but still could be possible, if we wanted to change files outside the output directory, such as the `.asf.yaml` file in those branches).